readme.md

```diff
@@ -0,0 +1,97 @@
+![](media/banner.png)
+
+<div align="center">
+  <a aria-label="Vercel logo" href="https://vercel.com">
+    <img src="https://img.shields.io/badge/made%20by-vercel-%23000000">
+  </a>
+  <br>
+  <a aria-label="Install Size" href="https://packagephobia.now.sh/result?p=serve">
+    <img src="https://packagephobia.now.sh/badge?p=serve">
+  </a>
+  <a aria-label="Stars" href="https://github.com/vercel/serve/stargazers">
+    <img src="https://img.shields.io/github/stars/vercel/serve">
+  </a>
+  <a aria-label="Build Status" href="https://github.com/vercel/serve/actions/workflows/ci.yaml">
+    <img src="https://github.com/gamemaker1/serve/actions/workflows/ci.yaml/badge.svg">
+  </a>
+</div>
+
+---
+
+`serve` helps you serve a static site, single page application or just a static file (no matter if on your device or on the local network). It also provides a neat interface for listing the directory's contents:
+
+![Listing UI](media/listing-ui.png)
+
+> Once it's time to push your site to production, we recommend using [Vercel](https://vercel.com).
+
+## Usage
+
+The quickest way to get started is to just run `npx serve` in your project's directory.
+
+If you prefer, you can also install the package globally using [Yarn](https://yarnpkg.com/en/) or [PNPM](https://pnpm.io/) (you'll need at least [Node LTS](https://github.com/nodejs/Release#release-schedule)):
```

Comment:
I don't think we need to list every package manager. Lets just list `npm i -g serve` and the others can be assumed.

Reply:
Okay, will revert.